core/deployment/src/main/java/io/quarkus/deployment/dev/DevModeMain.java

```diff
@@ -201,19 +196,8 @@ private void linkDotEnvFile() {
             silentDeleteFile(link);
             try {
                 // create a symlink to ensure that user updates to the file have the expected effect in dev-mode
-                try {
-                    Files.createSymbolicLink(link, dotEnvPath);
-                } catch (FileSystemException e) {
-                    // on Windows fall back to mklink if symlink cannot be created via Files API (due to insufficient permissions)
-                    // see https://github.com/quarkusio/quarkus/issues/8297
-                    if (SystemUtils.IS_OS_WINDOWS) {
-                        log.debug("Falling back to mklink on Windows after FileSystemException", e);
-                        makeHardLinkWindowsFallback(link, dotEnvPath);
-                    } else {
-                        throw e;
-                    }
-                }
-            } catch (IOException | InterruptedException e) {
```

Comment:
How come this part was removed?

Reply:
This code was apparently intended to work around https://bugs.openjdk.org/browse/JDK-8221852 which has been fixed since JDK 13.